helpers/test/test_hcache_simple.py

```diff
@@ -121,6 +121,10 @@ def set_up_test(self) -> None:
          - Reset in-memory caches for all cached functions.
          - Set specific cache properties needed for the tests.
         """
+        import helpers.hsystem as hsystem
+
+        # turn off the “are you sure?” prompt
+        hsystem.query_yes_no = lambda *args, **kwargs: False
```

Comment:
Ugh. Better to pass a param to reset_disk_cache like `interactive: bool = True` and then you pass False